pkg/k8s/service.go

```diff
@@ -162,6 +162,11 @@ func (c *Client) WaitForService(ctx context.Context, name string) error {
 	}
 }
 
+func (c *Client) ServiceDNS(name string) string {
+	return fmt.Sprintf("%s.%s.svc.cluster.local", name, c.namespace)
+}
```

Comment:
@smuu how useful do you think making it configurable would be?

Reply:
If using a default Kubernetes setup, this will not be an issue. There ar setups where the cluster domain is different.
One option would be to add this as an optional parameter of the knuu instance with `cluster.local` as the default value.

Reply:
that's a good idea. thanks guys, will add it.